include/statsdcc/configs/aggregator_config.h

```diff
@@ -33,6 +33,11 @@ class AggregatorConfig : public Config {
 
   std::string prefix;
 
+  std::string prefixCounter;
```

Comment:
To keep it consistent with the rest of the code, can we please stick with snake_case for variable names, please.

An example: [current_ledger](https://github.com/wayfair/statsdcc/blob/0cffedcebf93aff189fca5e8b6605d2e979e6cb2/include/statsdcc/workers/aggregator/worker.h#L119)

Reply:
Fixed.